.github/workflows/CI.yml

```diff
@@ -45,14 +45,20 @@ jobs:
       - uses: julia-actions/setup-julia@v1
         with:
           version: '1'
+      - run: sudo apt-get update && sudo apt-get install -y xorg-dev mesa-utils xvfb libgl1 freeglut3-dev libxrandr-dev libxinerama-dev libxcursor-dev libxi-dev libxext-dev
       - run: |
           julia --project=docs -e '
             using Pkg
             Pkg.develop(PackageSpec(path=pwd()))
+            Pkg.add(PackageSpec("GLMakie"))
+            Pkg.add(PackageSpec("GraphViz"))
+            Pkg.add(PackageSpec("Cairo"))
+            Pkg.add(PackageSpec("FileIO"))
```

Comment:
Why can't these dependencies be declared on the `Project.toml` of the `docs`? I think I missed something in the previous discussions.

Reply:
You're absolutely right, there's no reason not to add these to the `docs` environment.

More generally, I think we should make it easy for developers using `DataFlowTasks` to install these optional deps in a clean way (maybe not in their global environment). If we do this, the `docs` environment (and maybe `test` too) could benefit from that.

Laurent & I were thinking about making a subdirectory somewhere defining a proper environment for these optional deps. We should discuss this today